src/screens/EventDetailsScreen/component.js

```diff
@@ -166,20 +163,16 @@ class EventDetailsScreen extends PureComponent<Props> {
                       {event.fields.accessibilityDetails && <SectionDivider />}
                       {event.fields.accessibilityDetails && (
                         <EventAccessibility>
-                          {event.fields.accessibilityDetails[locale]}
+                          {event.fields.accessibilityDetails}
                         </EventAccessibility>
                       )}
                       {(event.fields.email || event.fields.phone) && (
                         <SectionDivider />
                       )}
                       {(event.fields.email || event.fields.phone) && (
                         <EventContact
-                          email={
-                            event.fields.email && event.fields.email[locale]
-                          }
-                          phone={
-                            event.fields.phone && event.fields.phone[locale]
-                          }
+                          email={event.fields.email && event.fields.email}
+                          phone={event.fields.phone && event.fields.phone}
```

Comment:
We should be able to remove these checks and just pass down `event.fields.email` and `event.fields.phone`.